pgxn/neon/libpagestore.c

```diff
@@ -1321,6 +1326,16 @@ pg_init_libpagestore(void)
 							PGC_USERSET,
 							0,	/* no flags required */
 							NULL, (GucIntAssignHook) &readahead_buffer_resize, NULL);
+	DefineCustomIntVariable("neon.readahead_getpage_pull_timeout",
+							"readahead response pull timeout",
+							"Time between active tries to pull data from the "
+							"PageStream connection when we have pages which "
+							"were read ahead but not yet received.",
+							&readahead_getpage_pull_timeout_ms,
+							-1, -1, INT_MAX,
+							PGC_USERSET,
+							GUC_UNIT_MS,	/* no flags required */
```

Comment:
```suggestion
							GUC_UNIT_MS,
```

GUC_UNIT_MS is a flag